daemon/lvm_utils.ml

```diff
@@ -34,15 +34,26 @@ open Utils
  * resolved.
  *)
 let lv_canonical device =
-  let stat1 = stat device in
-  let lvs = Lvm.lvs () in
-  try
-    Some (
-      List.find (
-        fun lv ->
-          let stat2 = stat lv in
-          stat1.st_rdev = stat2.st_rdev
-      ) lvs
-    )
-  with
-  | Not_found -> None
+  let stat1 =
+    try Some (stat device)
+    with Unix_error _ -> None in
```

Comment:
Hiding all errors like this is usually a problem.  It could, for example, hide I/O errors when we definitely want those to escape.  It's better to hide the single error (probably `ENOENT`?) that you saw.